### ISSUE TITLE
Cannot Import Account on Scatter extension on Chrome

### ISSUE THREAD

**Issue by Stathismi** [EOSEssentials/Scatter#117]
i create a key pair for EOS , I give it a name , then go on identity to import an account , select it and get that no ACCOUNTS FOUND. how can i deal with this? thanks
<img width="311" alt="screen shot 2018-06-25 at 15 59 41" src="https://user-images.githubusercontent.com/36051231/41860093-a2703ace-7895-11e8-9ddb-2dbabab81130.png">

**Comment by nsjames** [comment]
Please screenshot you networks panel ( settings -> networks )

**Comment by chiliec** [comment]
<img width="361" alt="screen shot 2018-06-28 at 21 17 42" src="https://user-images.githubusercontent.com/982358/42047069-bb28c0b0-7b18-11e8-91b4-cf7d0c56e5cb.png">
Same thing.
https://nodes.get-scatter.com not available :(

**Comment by chiliec** [comment]
Oh no... It's restricted only from russian IP. But why?

**Comment by nsjames** [comment]
Yeah seems so, turn on a VPN and import your account. You wont need to keep turning on VPN to use Scatter though.
